### PR TITLE
Add regional flag

### PR DESCRIPTION
**Pull request by aleksandra-malinowska**
Use flag to identify regional clusters iinstead of relying on multi-zone property.

### CODE REVIEW

File: cluster-autoscaler/cloudprovider/gce/gce_manager.go [hunk 1]
```diff
@@ -220,7 +218,7 @@ func CreateGceManager(configReader io.Reader, mode GcpCloudProviderMode, cluster
 		gceService:  gceService,
 		migCache:    make(map[GceRef]*Mig),
 		location:    location,
-		isRegional:  isRegional,
+		isRegional:  regional,
```
Comment (bskiba) [inline cluster-autoscaler/cloudprovider/gce/gce_manager.go]: nit: let's make this consistent and go for one of isRegional and regional in both places.

**Comment by bskiba** [comment]
One small comment, otherwise lgtm

File: cluster-autoscaler/cloudprovider/gce/gce_manager.go (hunk 1 above)
Comment (aleksandra-malinowska) [reply cluster-autoscaler/cloudprovider/gce/gce_manager.go]: Done. Went with `regional` following the AllElseBeingEqualLessCamelCaseIsBetter principle;)

**Comment by negz** [comment]
Just curious - what was the reasoning behind this change? Is the multi-zone config knob being removed?

**Comment by bskiba** [comment]
The two flags have slightly different meanings so they'll both stay: 
multi-zone - the worker nodes may be in multiple zones
regional - master nodes may be in multiple zones

So for example these two configs are valid and different from autoscaler point of view:
1) --multi-zone=true --regional=false (nodes across zones but master in just one zone)
2) --multi-zone=true --regional=true (nodes across zones and master replicated across zones)

**Pull request merged**